### PR TITLE
Crs 1785: Make the typescript generation work

### PR DESCRIPTION
**Pull request by dave-virden**

### CODE REVIEW

File: server/services/calculateReleaseDatesService.test.ts [hunk 1]
```diff
@@ -419,21 +422,6 @@ describe('Calculate release dates service tests', () => {
         { hintText: '20 March 2023 plus 66 days', releaseDate: '2023-03-20', releaseDateType: 'ERSED' },
       ])
     })
-    it('ERSED one year', async () => {
```
Comment (justin-gardiner) [inline server/services/calculateReleaseDatesService.test.ts:422]: is this test mean to be removed?

File: server/services/calculateReleaseDatesService.ts [hunk 2]
```diff
@@ -273,16 +273,6 @@ export default class CalculateReleaseDatesService {
     unadjustedDate: string,
     adjustedDays: number,
   ): ReleaseDateWithAdjustments {
-    if (rules.includes('ERSED_ONE_YEAR')) {
```
[the remark this answers is not in the document]
Comment (dave-virden) [reply server/services/calculateReleaseDatesService.ts:276]: I think so, discussed it last thursday, see 2.43pm onwards https://mojdt.slack.com/archives/C027NTF1022/p1706712188571409

[the remark this answers is not in the document]
Comment (justin-gardiner) [reply server/services/calculateReleaseDatesService.ts:276]: LGTM

**Review by justin-gardiner — APPROVED** [review]
LGTM

**Pull request merged**